packages/sdk/src/resolve/resolve-cadence.js

```diff
@@ -1,20 +1,42 @@
 import {isTransaction, isScript, get} from "../interaction/interaction.js"
 import {invariant} from "@onflow/util-invariant"
 import {config} from "@onflow/config"
+import * as logger from "@onflow/util-logger"
 
 const isFn = v => typeof v === "function"
 const isString = v => typeof v === "string"
 
+const accountIdentifierPattern = /\b(0x\w+)\b/g
+function isAccountIdentifierSyntax(cadence) {
+  return accountIdentifierPattern.test(cadence)
+}
+
+const contractIdentifierPatternFn = () => /import\s+"(\w+)"/g
+export function isContractIdentifierSyntax(cadence) {
```

Comment:
Same thing on test here

Reply:
We are testing the exported functions of the file which should in turn exercise that function.